scripts/development/m3_stack/start_m3.sh

```diff
@@ -9,9 +9,21 @@ if [[ "$FORCE_BUILD" = true ]] ; then
     DOCKER_ARGS="--build -d --renew-anon-volumes"
 fi
 
+# need to start Jaeger before m3db or else m3db will not be able to talk to the Jaeger agent.
+if [[ "$START_JAEGER" = true ]] ; then
+    docker-compose -f docker-compose.yml up $DOCKER_ARGS jaeger
+    sleep 5
+    JAEGER_STATUS=$(curl -s -o /dev/null -w '%{http_code}' localhost:14269)
+    if [ $JAEGER_STATUS -ne 204 ]; then
```

Comment:
Once https://github.com/jaegertracing/jaeger/issues/1450 is fixed, we can change this logic.

Reply:
nit: for posterity, could you put in a comment saying the same thing in the file

Reply:
Perhaps a nit, but is there not an explicit way to state the dependency in `docker-compose.yml`? https://docs.docker.com/compose/compose-file/ mentions `depends_on`